### PR TITLE
[14.0][MIG] l10n_th_withholding_tax

### PR DESCRIPTION
**Pull request by Saran440**
In version14
- Invoices : type -> move_type 
- Register Payment Wizard : account.payment -> account.payment.register
- Test script can't download demo data (account_minimal_test.xml)

### CODE REVIEW

File: l10n_th_withholding_tax/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_withholding_tax,withholding.tax.cert,model_account_withholding_tax,account.group_account_invoice,1,1,1,1
```
Comment (kittiu) [inline l10n_th_withholding_tax/security/ir.model.access.csv]: No `cert`
(v13 already has this `cert`)

**Comment by Saran440** [comment]
Fixed. Thank you got your reviewed.

**Comment by kittiu** [comment]
@Saran440 please include this merged Fix.
https://github.com/OCA/l10n-thailand/pull/196/files

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
Rebased.

**Pull request merged**